### PR TITLE
[IMP] account: Add reversal move values prepare function

### PR DESCRIPTION
**Pull request by rousseldenis**
Nothing changed in behavior.

@william-andre @Whenrow 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account_move.py [hunk 1]
```diff
@@ -3339,22 +3347,15 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
 
         reverse_moves = self.env['account.move']
         for move, default_values in zip(self, default_values_list):
-            default_values.update({
-                'move_type': TYPE_REVERSE_MAP[move.move_type],
-                'reversed_entry_id': move.id,
-                'partner_id': move.partner_id.id,
-            })
+            move._prepare_reversal_values(default_values)
```
Comment (william-andre) [inline addons/account/models/account_move.py:3350]: This seems useless, you can already add the values by inheriting `_reverse_moves` and zipping the list with self

File: addons/account/models/account_move.py [hunk 2]
```diff
@@ -3339,22 +3347,15 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
...
                 include_business_fields=True,
                 skip_invoice_sync=move.move_type == 'entry',
             ).copy(default_values)
 
         reverse_moves.with_context(skip_invoice_sync=cancel).write({'line_ids': [
-            Command.update(line.id, {
-                'balance': -line.balance,
-                'amount_currency': -line.amount_currency,
-            })
+            Command.update(line.id, line._prepare_reversal_values())
```
Comment (william-andre) [inline addons/account/models/account_move.py:3358]: Why would you need anything in here?
This only applies to misc operations, where the only thing that needs to be done is reverse the amounts; everything else needs to be exactly the same.

**Review by william-andre — COMMENTED** [review]
Sorry for the late review, this was lost in my emails

Would you mind giving an example why you need this in practice?
Like always, I try not to add hooks that seem useless and only make the code harder to read, or allow doing wrong stuff.

**Comment by qdp-odoo** [comment]
looks like no one really care. :man_shrugging: 

Closing it, for the sake of "spring cleaning"

**Pull request closed without merging**